src/util.js

```diff
@@ -10,6 +10,28 @@ WaveSurfer.util = {
         return dest;
     },
 
+    min: function(values) {
+        var min = null;
```

Comment:
The initial value could be `Infinity`, then you don't need to check if it's null.